cincinnati/src/lib.rs

```diff
@@ -75,12 +75,16 @@ impl Release {
     }
 
     /// Returns the `manifestref` of a given `Release`
-    pub fn manifestref(&self) -> Result<&String, Error> {
+    pub fn manifestref(&self) -> Result<String, Error> {
         let digestkey = String::from("io.openshift.upgrades.graph.release.manifestref");
         match self {
             Release::Concrete(release) => {
                 let digest = release.metadata.get(&digestkey);
-                Ok(digest.map(|d| d).unwrap())
+                let manifestref = match digest.map(|d| d) {
+                    Some(manifestref) => manifestref.clone(),
+                    None => String::from("no manifestref for release"),
```

Comment:
If all the logic expects a valid `manifestref` but some `ConcreteRelease` do not have one, it means that either 1) the design/assumption is invalid, or 2) the input was invalid and it has not been properly validated.
Creating a fake arbitrary string here won't do any good to the situation.

Is a release without a manifestref a valid one? If yes, then this method should reflect that through an `Option<_>` result. If not, then the input parsing logic should be enhanced to detect and highlight the invalid input.

Getting rid of this `unwrap` would be overall good, but likely it should become an `ok_or_else(...)?`.